chef-utils/lib/chef-utils/dist.rb

```diff
@@ -64,6 +64,9 @@ class Infra
 
       # The client's gem
       GEM = "chef"
+
+      # The client's container image
+      CONTAINER_IMAGE = "chef/chef"
```

Comment:
Could we use CONTAINER_REF = "chef/chef:latest" instead?

Reply:
Could we use something like:  "chef/chef:latest" instead? - Probably not since this could cause issue with the way it gets used in kitchen.

Reply:
Would it be possible to provide some context around where this would be used?

Reply:
@PrajaktaPurohit  The intent is to replace this explicit reference with this DIST reference https://github.com/test-kitchen/kitchen-dokken/blob/main/lib/kitchen/driver/dokken.rb#L40

So then the Cinc project can replace this value in the downstream release of Cinc Workstation to make for a seamless transition when utilizing Kitchen-Dokken.

Reply:
> Could we use CONTAINER_REF instead of CONTAINER_IMAGE.

Do you still want this change? I'm fine either way

Reply:
Yes, please make this update

Reply:
Updated